### PR TITLE
Sbi 17 multi site blog post

### PR DESCRIPTION
**Pull request by maggyb**

### CODE REVIEW

File: _posts/2016-10-27-journal-of-a-junior-multisite.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+layout:         post
+title:          "Journal of a Junior: Using one Umbraco instance to host multiple sites"
+date:           2016-10-27
+author_id:      "maggy_burns"
+description:    "Hello, Maggy here! Here's my second blog post in my Journal of a Junior Series. A tutorial on how to host multiple sites on one Umbraco instance."
+categories:     space between
+banner:         "/assets/images/blog/tutoring.jpeg"
+---
+
+Hello! Maggy here on my long overdue second blog post in my Journal of a Junior series! This is my sixth month working at Space Between, I have worked on so many projects, with so many amazing technologies it was hard to pick what to write this post on. I’ve decided to go with using one Umbraco CMS to host multiple sites!
```
Comment (lukefrake) [inline _posts/2016-10-27-journal-of-a-junior-multisite.md]: One Umbraco CMS _instance_

Comment (lukefrake) [reply _posts/2016-10-27-journal-of-a-junior-multisite.md]: I'd probably link to Umbraco website on this

File: _posts/2016-10-27-journal-of-a-junior-multisite.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+
+I've chosen this because it was one of the tasks I thought was going to be very difficult. It involved so many things I'd never encountered before, and was a big learning curve - but very rewarding and now something that I realise can be done in a flash! This tutorial will assume you are familiar with the Umbraco CMS and being able set up different domain names in IIS for each site.
+
+
+###Step 1: Setting up
```
Comment (lukefrake) [inline _posts/2016-10-27-journal-of-a-junior-multisite.md:18]: I would suggest starting at Creating DocTypes and just mention in it that people can see the documentation page for getting started.

This section doesn't really say anything about getting started.

File: _posts/2016-10-27-journal-of-a-junior-multisite.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+author_id:      "maggy_burns"
+description:    "Hello, Maggy here! Here's my second blog post in my Journal of a Junior Series. A tutorial on how to host multiple sites on one Umbraco instance."
+categories:     space between
+banner:         "/assets/images/blog/tutoring.jpeg"
+---
+
+Hello! Maggy here on my long overdue second blog post in my Journal of a Junior series! This is my sixth month working at Space Between, I have worked on so many projects, with so many amazing technologies it was hard to pick what to write this post on. I’ve decided to go with using one Umbraco CMS to host multiple sites!
+
+
+I've chosen this because it was one of the tasks I thought was going to be very difficult. It involved so many things I'd never encountered before, and was a big learning curve - but very rewarding and now something that I realise can be done in a flash! This tutorial will assume you are familiar with the Umbraco CMS and being able set up different domain names in IIS for each site.
```
Comment (lukefrake) [inline _posts/2016-10-27-journal-of-a-junior-multisite.md:15]: I love this paragraph 😃

File: _posts/2016-10-27-journal-of-a-junior-multisite.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+Umbraco CMS is open-source, free to use and also comes with a great community of developers. Here at Space Between we are big fans - so much that myself and Marcus did an Umbraco course and are now Certified Umbraco Developers!
+
+
+For those who haven't used Umbraco before, you can get your installation [here](https://our.umbraco.org/documentation/Getting-Started/Setup/Install/).
+
+
+###Step 2: Creating DocTypes
+
+
+I'm going to be showing the multisite we made for [HRGO Recruitment](http://www.hrgo.co.uk/) in this example. You will want to start with creating some templates for your sites. I personally have used the same templates for each different site to save a huge library of unique templates. I prefer having reusable templates and this is very easy with the use of the Umbraco Grid and special grid editors. I won't go into this too much, as there are many resources on the web for this.
```
Comment (lukefrake) [inline _posts/2016-10-27-journal-of-a-junior-multisite.md]: If I was reading this I'd love to read some example resources.

**Review by lukefrake — CHANGES_REQUESTED** [review]
Reeeeally good post Maggy, great work :)

couple of minor points to look at, but really good!

File: _posts/2016-10-27-journal-of-a-junior-multisite.md (hunk 2 above)
Comment (lukefrake) [reply _posts/2016-10-27-journal-of-a-junior-multisite.md:18]: Step 1 goes to step 3 now.

**Review by lukefrake — APPROVED** [review]

**Comment by lukefrake** [comment]
👍

File: _posts/2016-10-27-journal-of-a-junior-multisite.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+      return currentHome.Children.Where(x => x.IsDocumentType("settings")).First();
+}
+</xmp>
+
+And Razor to assign the stylesheet to the template:
+<xmp>
+@{
+    var websiteSettings = SettingsHelpers.GetWebsiteSettings();
+    var styleSheet = websiteSettings.GetProperty("styleSheet").Value;
+    string fonts = websiteSettings.GetProperty("fontReference").Value.ToString();
```
Comment (f000) [inline _posts/2016-10-27-journal-of-a-junior-multisite.md:80]: btw I think more elegant style would be websiteSettings.GetPropertyValue<string>("fontReference");

File: _posts/2016-10-27-journal-of-a-junior-multisite.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+[content]: /assets/images/blog/ums-content-tree.png "Content Tree"
+
+
+Now give each site its own unique content! If each site is having its own stylesheet or fonts, then this is where you will add them into the settings. First, you will want to make your stylesheets have been created. In the settings, you can add the file name and your link to the font you have chosen. I'm using google fonts, so I can just add the google link but this can easily be customised to suit you.
+
+
+We used a helper class to get the website settings node:
+<xmp>
+public static IPublishedContent GetWebsiteSettings()
+{
```
Comment (f000) [inline _posts/2016-10-27-journal-of-a-junior-multisite.md:66]: btw idk if we should present this piece of code... I think it's suboptimal in terms of caching

**Pull request merged**